src/include/daos/event.h

```diff
@@ -87,6 +87,10 @@ daos_event_complete(daos_event_t *ev, int rc);
 int
 daos_event_launch(struct daos_event *ev);
 
+/** convert event error to positive errno instead of -DER on completion */
```

Comment:
I'm not familiar with the event code, but do we ONLY want to convert to errno on **_completion_**?
I.e. we don't want to convert to errno for these function calls?
- daos_event_launch
- daos_event_register_comp_cb
- daos_event_parent_barrier
- (Maybe others)

Reply:
right the idea is to just convert the errno of the **event** itself, not the event API. this will always return -DER and that can be converted by users of that API.
we want to make the DFS API be uniform in what it returns. this is not about the event API iteself

Reply:
Okay, that makes sense. Thanks!